### PR TITLE
Fix require paths

### PR DESCRIPTION
**Pull request by timrourke**
This PR removes the `./lib/` prefix to `require` paths. They are not necessary, and cause problems when running the actual application.

### CODE REVIEW

File: lib/applicants/applicant_profile_presenter.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 require "./app/validators/url_validator"
```
Comment (bartboy011) [inline lib/applicants/applicant_profile_presenter.rb]: Does this need to be updated as well?

File: spec/footprints-memory-repository/note_repository_spec.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
-require './lib/memory_repository/note_repository'
+require 'memory_repository/note_repository'
 require './spec/footprints/shared_examples/note_examples'
```
Comment (bartboy011) [inline spec/footprints-memory-repository/note_repository_spec.rb]: Same here, does this need to be updated?

File: spec/footprints-memory-repository/user_repository_spec.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
-require './lib/memory_repository/user_repository'
+require 'memory_repository/user_repository'
 require './spec/footprints/shared_examples/user_examples'
```
Comment (bartboy011) [inline spec/footprints-memory-repository/user_repository_spec.rb]: And here?

File: lib/applicants/applicant_profile_presenter.rb (hunk 1 above)
Comment (timrourke) [reply lib/applicants/applicant_profile_presenter.rb]: hesitant to change that until i see an issue pop up, do you expect this to create problems as well?

Comment (bartboy011) [reply lib/applicants/applicant_profile_presenter.rb]: Let's assume it's good for now

**Pull request merged**